src/platform/common/utils/localize.ts

```diff
@@ -214,25 +238,28 @@ export namespace DataScience {
 
     export const dataExplorerTitle = localize('DataScience.dataExplorerTitle', 'Data Viewer');
     export const badWebPanelFormatString = localize(
-        'DataScience.badWebPanelFormatString',
+        {
+            key: 'DataScience.badWebPanelFormatString',
+            comment: ['{RegEx=">[^<]*<"}', 'Only translate the text within the HTML tags']
```

Comment:
This RegEx seems ok to me, but I don't trust my eyes with RegExes 😆 please let me know if it has a mistake 🙏

Reply:
You can test it with regex101.com.